### ISSUE TITLE
Beta directions for immune related traits are incorrect

### ISSUE THREAD

**Issue by edm1** [opentargets/issues#2077]
Beta directions for a number of immune traits including de Lange (GCST004131) or Liu (GCST003043) are incorrect

As can be seen in the GWAS Coloc table here:
https://dev--genetics-app.netlify.com/study-locus/GCST003044/2_233252937_T_C

**Comment by d0choa** [comment]
@buniello when you have a few minutes, could you confirm genetics portal looks in line with GWAS Catalog?

**Comment by buniello** [comment]
@d0choa: yes, triaged on a number of variants from the three De Lange studies and genetics portal betas are now in line with GWAS Catalog harmonised summary statistics

**Comment by d0choa** [comment]
Thanks for you help @buniello!
